bigtop-packages/src/charm/kafka/layer-kafka/tests/01-deploy.py

```diff
@@ -33,9 +33,19 @@ def setUpClass(cls):
         cls.d.configure('openjdk', {'java-type': 'jdk',
                                     'java-major': '8'})
 
-        cls.d.relate('kafka:zookeeper', 'zk:zkclient')
+        try:
```

Comment:
I believe that the old relation is "zkclient". The new relation is "zookeeper", which matches the name in the promulgated apache-zookeeper.

Basically, we decided not to be backwards compatible before my time :-p

I can change the new relation to be zkclient if you want, though.